src/main/java/seedu/eduke8/Eduke8.java

```diff
@@ -24,14 +26,15 @@
     private static final String LOG_PATH = "data/logs/" + DATE_TIME_NOW.format(DATE_TIME_FORMATTER) + ".log";
     private static final Logger LOGGER = Logger.getLogger(Logger.GLOBAL_LOGGER_NAME);
 
+    public BookmarkList bookmarks = new BookmarkList();
     private MenuParser menuParser;
     private TopicsStorage topicsStorage;
     private LogStorage logStorage;
     private TopicList topicList;
     private Ui ui;
 
     private Eduke8(String dataPath, String logPath) {
-        menuParser = new MenuParser();
+        menuParser = new MenuParser(bookmarks);
```

Comment:
this one as well may have to create a `User` class to include notes next time but for now it seems fine